test/edu-benefits/1990-rjsf/config/employmentHistory.unit.spec.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990e/config/form';
+
+describe('Edu 1990e employmentHistory', () => {
+  const { schema, uiSchema } = formConfig.chapters.employmentHistory.pages.employmentHistory;
+  const definitions = formConfig.defaultDefinitions;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}
+          definitions={definitions}/>
+    );
+
+    const inputs = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input');
```

Comment:
Nope, I mixed up the 1990 and 1990e tests. I meant to update both.

I don't think there's any good reason to use `scryRenderedDOMComponentsWithTag` now. I think it's just a decision about whether to update it or not.